src/App.tsx

```diff
@@ -1,16 +1,68 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number;
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state = {
```

Comment:
```suggestion
  state: State = {
```
better to write it explicitly